src/Ads/AssetSuggestionsService.php

```diff
@@ -17,13 +19,181 @@
  */
 class AssetSuggestionsService implements Service {
 
+	/**
+	 * Default maximum marketing images.
+	 */
+	protected const DEFAULT_MAXIMUM_MARKETING_IMAGES = 20;
+
 	/**
 	 * AssetSuggestionsService constructor.
 	 *
-	 * @param WP $wp
+	 * @param WP $wp WP Proxy.
+	 * @param WC $wc WC Proxy.
 	 */
-	public function __construct( WP $wp ) {
+	public function __construct( WP $wp, WC $wc ) {
 		$this->wp = $wp;
+		$this->wc = $wc;
+	}
+
+	/**
+	 * Get assets from specific post or term.
```

Comment:
💅 Maybe it'd be clearer if we rephrase it as `Get WP and other campaigns' assets from the specific post or term.`?